src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> Why don't you want to use something that we already have (strings.xml)? 

I am not against it as (if you remember), I suggested it initially in one of the issues. `rnpm.xml` was just an idea I had in my mind as a drop-in replacement for `moduleConfig` (which in fact should be named `rnpmConfig` to be more explicit about it source). 

> Which issues would we have if we use strings.xml? Which benefits rnpm.xml brings to the table?

Please read carefully what I wrote in previous comments. Apart from that, when we bring more types to the table (and I think bools are quite cool and I seen them in few modules), if you go with `strings.xml`, you are most likely to create `bools.xml`, `arrays.xml` and other file types (which is actually cool, because it follows `android` conventions) - however, in order to get all keys managed by `rnpm`, you will have to actually read, parse and concat arrays from multiple files. It's mainly a design decision regarding how we see extra parameters working in the future.

> It's a system parameter that mark strings

I don't think neither of these are correct, but I get what you mean.